src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleWidget.ts

```diff
@@ -41,13 +41,15 @@ export abstract class TerminalAccessibleWidget extends DisposableStore {
 	protected _listeners: IDisposable[] = [];
 
 	private readonly _focusedContextKey?: IContextKey<boolean>;
+	private readonly _focusedLastLineContextKey?: IContextKey<boolean>;
 	private readonly _focusTracker?: dom.IFocusTracker;
 
 	constructor(
 		private readonly _className: string,
 		protected readonly _instance: Pick<ITerminalInstance, 'shellType' | 'capabilities' | 'onDidRequestFocus' | 'resource'>,
 		protected readonly _xterm: Pick<IXtermTerminal, 'shellIntegration' | 'getFont'> & { raw: Terminal },
 		private _focusContextKey: RawContextKey<boolean> | undefined,
+		private _focusLastLineContextKey: RawContextKey<boolean> | undefined,
```

Comment:
The names of `_focusContextKey` and `_focusedContextKey` are confusing. I suggest `_focusContextKey` and `rawFocusContextKey` (no `private _` as it's only used in the ctor)